src/Microsoft.OData.Core/JsonLight/ODataJsonLightOutputContext.cs

```diff
@@ -42,8 +42,13 @@ internal sealed class ODataJsonLightOutputContext : ODataOutputContext
         /// <summary>The message output stream.</summary>
         private Stream messageOutputStream;
 
+        #if NETSTANDARD1_1
         /// <summary>The asynchronous output stream if we're writing asynchronously.</summary>
         private AsyncBufferedStream asynchronousOutputStream;
+        #else
+	    /// <summary>The asynchronous output stream if we're writing asynchronously.</summary>
+	    private Stream asynchronousOutputStream;

```

Comment:
I'm not sure how strict we are in this repo about formatting, but there seems to be consistent formatting issues. I'm not sure if it's tabs vs spaces or due to the compiler directives or what...